ert3/console/_console.py

```diff
@@ -62,6 +62,11 @@ def valid_record_file(path: Union[str, Path]) -> Path:
         type=valid_record_file,
         help="Path to resource file",
     )
+    record_load_parser.add_argument(
+        "--record-type",
+        help="Record type blob or normal (default: normal)",
+        default="normal",
```

Comment:
Or even just `--blob-record`.

Reply:
My initial thought was to support more types in the future. But good comments both

Reply:
We might end up with more types, but it is not entirely obvious. And since we are not yet aiming for a stable interface, perhaps we can just get started with the suggestion by @jondequinor 👍🏻